source/ShopPanel.h

```diff
@@ -30,8 +30,6 @@ class Planet;
 class PlayerInfo;
 class Ship;
 
-
-
```

Comment:
Some styling lines got nixed when the enum was moved in throughout header; please restore them

Reply:
Thank you, restored.